src/js/model/index.js

```diff
@@ -24,16 +23,8 @@ var pipelines = [],
     primitives = {},
     listeners = {};
 
-/**
- * Initializes the Lyra model with a new Scene primitive.
- * @returns {Object} The Lyra Model
- */
-model.init = function() {
-  var Scene = require('./primitives/marks/Scene');
-  model.Scene = new Scene().init();
-  store.dispatch(expandLayers([model.Scene._id]));
-  return this;
-};
+window.listeners = listeners;
+window.primitives = primitives;
```

Comment:
yeaaah one of my todos tonight is to audit all of the stray debugging code; i'm still usin' 'em tho so i it's OK for you can they live where they are through this merge? :D